### PR TITLE
Tidy PR template

### PR DESCRIPTION
**Pull request by kwankyu**
Our current PR template makes one to think and search for the blank where to put the description. We tidy the PR template so that it is obvious where the blank is. Remove other less useful lines. 

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,32 +1,26 @@
...
 
 ### :memo: Checklist
 
 <!-- Put an `x` in all the boxes that apply. -->
-<!-- If your change requires a documentation PR, please link it appropriately -->
-<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-<!-- Feel free to remove irrelevant items. -->
 
-- [ ] The title is concise, informative, and self-explanatory.
+- [ ] The title is concise, informative.
```
Comment (grhkm21) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] The title is concise and informative.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,32 +1,26 @@
...
-<!-- Why is this change required? What problem does it solve? -->
-<!-- If this PR resolves an open issue, please link to it here. For example "Fixes #12345". -->
-<!-- If your change requires a documentation PR, please link it appropriately. -->
 
 ### :memo: Checklist
 
 <!-- Put an `x` in all the boxes that apply. -->
-<!-- If your change requires a documentation PR, please link it appropriately -->
-<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-<!-- Feel free to remove irrelevant items. -->
```
Comment (grhkm21) [inline .github/PULL_REQUEST_TEMPLATE.md:17]: Is there a strong reason why these two lines should be removed?

**Review by grhkm21 — CHANGES_REQUESTED** [review]
Let's get this done, it's a good change :D

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (kwankyu) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: That is about the checklist. 

First line: The items of the checklist seem clear enough. I doubt that someone wants to ask about them. 

Second line: No one needs to remove items from the checklist. They are checked or left unchecked.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (kwankyu) [reply .github/PULL_REQUEST_TEMPLATE.md]: Thanks. Done.

**Comment by grhkm21** [comment]
Thanks! Looks good

**Comment by kwankyu** [comment]
Thank you!

**Comment by kwankyu** [comment]
Sorry for late commit. I removed one blank line, which seems inserted by accident.

**Comment by grhkm21** [comment]
Still looks good to me, sorry for not spotting it :)

**Pull request merged**